repository/Zinc-Seaside.package/ZnZincServerAdaptor.class/instance/responseFrom..st

```diff
@@ -1,6 +1,9 @@
 converting
 responseFrom: aRequestContext
 	| partialHeaders cookies fullHeaders seasideResponse contents entity contentType |
+	"TODO: is websocket only... useful/needed to package in separate package?"
+	(aRequestContext properties at: #webSocket ifAbsent:[ nil ])
```

Comment:
I noticed the formatting in a lot of places omits a space after `:`. Is this intentional?

Reply:
I have a lot to clean up still and there’s a lot of copy paste for now. I made a draft PR to follow up on the CI build. So most of this code is not in the final clean state, rather fast prototyping.